spec/03-plugins/03-http-log/01-log_spec.lua

```diff
@@ -269,4 +312,67 @@ for _, strategy in helpers.each_strategy() do
       end, 10)
     end)
   end)
-end
+
+  describe("Plugin: http-log (log) [#" .. strategy .. "]", function()
```

Comment:
This describe block needs to play nicely with its parent block. As of now, this test’s name would be:

```
Plugin: http-log (log) [#postgres] Plugin: http-log (log) [#postgres] successfully executes gobally [...]
```

Something like this would be preferable:
```
Plugin: http-log (log) [#postgres] enables globallly executes successfully
```

With:
```
describe("enabled globally")
it("executes successfully")
```

This way, the test are properly scopes within a describe block about usage of this plugin when enabled globally, and the name is readable, plus the door is open for more `it()` test cases (e.g. `it("sets the name to 'global'")`)

Reply:
I think my last commit resolved this.